src/discordBot.py

```diff
@@ -24,84 +25,30 @@ def getEmbedObject(text, player, command):
 def removeTwitchEmotes(s):
     for key in emotes:
         s = s.replace(key, '')
-
     return s
 
-@bot.command()
-async def bgrank(ctx, *args):
-    args = args or ['lii']
-    args = args[:2]
-    args = list(args)
-
-    # Handle !bgrank EU for example
-    if parseRegion(args[0]):
-        region = parseRegion(args[0])
-        args = ['lii', region]
-
-    # Handle if a rank is passed in with a region
-    # if len(args) > 1:
-    #     args[0] = get_tag_from_rank(args[0], args[1])
-
-    response = removeTwitchEmotes(leaderboardBot.getRankText(*args))
-
+def call(ctx, func, name, *args):
+    response = removeTwitchEmotes(func(*args))
```

Comment:
May not need to removeTwitchEmotes twice as its called in the ctx.send call below.

Reply:
this was one of the places I wasn't sure on, do I need to make this an `async def`. Agree this only needs to be called once.